content/en/docs/ops/integrations/skywalking/index.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Apache SkyWalking
+description: How to integrate with Apache SkyWalking.
+weight: 32
+keywords: [integration,skywalking,tracing]
+owner: istio/wg-environments-maintainers
+test: no
+---
+
+[Apache SkyWalking](http://skywalking.apache.org) is an application performance monitor (APM) system, especially designed for
+microservices, cloud native and container-based architectures.
+
+## Installation
+
+### Option 1: Quick start
+
+Istio provides a basic sample installation to quickly get SkyWalking up and running:
+
+{{< text bash >}}
+$ kubectl apply -f @samples/addons/extras/skywalking.yaml@
+{{< /text >}}
+
+This will deploy SkyWalking into your cluster. This is intended for demonstration only, and is not tuned for performance or security.
+
+Istio proxies by default don't send traces to SkyWalking, you will also need to enable the SkyWalking tracing extension provider with
+the following fields to your configuration:
+
+{{< text yaml >}}
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  meshConfig:
+    extensionProviders:
+      - skywalking:
+          service: tracing.istio-system.svc.cluster.local
+          port: 11800
+        name: skywalking
+    defaultProviders:
+        tracing:
+        - "skywalking"
+{{< /text >}}
+
+### Option 2: Customizable install
+
+Consult the [SkyWalking documentation](http://skywalking.apache.org) to get started. No special changes are needed for SkyWalking to work with Istio.
+
+Once SkyWalking is installed, remember to modify the option `--set meshConfig.extensionProviders[0].skywalking.service` to point to the oap deployment.
```

Comment:
What's an oap?

Reply:
> What's an oap?

It's the collector of skywalking, I'll modify it to `skywalking-oap` because this is the deployment name